btcec/pubkey.go

```diff
@@ -63,13 +63,10 @@ func IsCompressedPubKey(pubKey []byte) bool {
 		return false
 	}
 
-	// Next, extract the format that's encoded within the first byte of the
-	// serialized pubkey.
-	format := pubKey[0]
```

Comment:
The length check above should be removed too since it's covered by this statement.